src/scripts/main.js

```diff
@@ -1,3 +1,80 @@
 'use strict';
 
-// write code here
+const buttonPlusRow = document.querySelector('.append-row');
+const buttonDelRow = document.querySelector('.remove-row');
+const buttonPlusCol = document.querySelector('.append-column');
+const buttonDelCol = document.querySelector('.remove-column');
```

Comment:
don't crop names, fix all similar cases
```suggestion
const deleteColumnButton = document.querySelector('.remove-column');
```